src/helpers/UserMessageError.ts

```diff
@@ -0,0 +1,6 @@
+export class UserMessageError extends Error {
```

Comment:
It occurs to me that the use of this error over a regular `Error` object should probs be documented. Just looking at this class definition, `UserMessageError` looks like a useless abstraction over `Error`, when in reality it's handled specially by the command invoker, and commands should prefer `UserMessageError` over other error types for intentional userland error messages.

Reply:
done